vars/signArtifacts.groovy

```diff
@@ -30,8 +30,9 @@ void call(Map args = [:]) {
 }
 
 void importPGPKey(){
+    keyPath = "$WORKSPACE/opensearch.pgp"
```

Comment:
Is this `keyPath` necessary? I think we are justing taking the `gpg import` out of the if bracket.

Reply:
Yeah, I realized that the same path is being used at multiple places. So it is better to have in a variable. This makes it less error prone (for any changes in the path in future) and easy to follow.